jvm/experiment.yaml

```diff
@@ -1,44 +1,41 @@
-apiVersion: redskyops.dev/v1alpha1
+apiVersion: redskyops.dev/v1beta1
 kind: Experiment
 metadata:
-  name:  jvm-reactors
+  name: jvm-reactors
 spec:
+  constraints:
+  - order:
+      lowerParameter: heap_size
+      upperParameter: memory
+  metrics:
+  - minimize: true
+    name: duration
+    port: 0
```

Comment:
I think we can drop this `port: 0` bit

Reply:
Yeah, I noticed this in other places as well: it's an artifact of dumping Kube output since `IntOrString` is not playing well with `omitempty`.